python/amici/pysb_import.py

```diff
@@ -188,14 +194,21 @@ def ode_model_from_pysb_importer(
     :param simplify:
             see :attr:`amici.ODEModel._simplify`
 
+    :param cache_simplify:
+            see :func:`amici.ODEModel.__init__`
+
     :param verbose: verbosity level for logging, True/False default to
         :attr:`logging.DEBUG`/:attr:`logging.ERROR`
 
     :return:
         New ODEModel instance according to pysbModel
     """
 
-    ode = ODEModel(verbose=verbose, simplify=simplify)
+    ode = ODEModel(
+        verbose=verbose,
+        simplify=simplify,
+        cache_simplify=cache_simplify,
```

Comment:
If it's known to fail for pysb models, I think we shouldn't offer it as an option, but rather set it to `False` and add a comment that it isn't compatible.

Reply:
Hm, now not sure. Just tried to debug the PySB issue by settings the default value in `pysb2amici` and `ode_model_from_pysb_importer` to `True`, but all tests in `pytest test_pysb.py` passed.

Reply:
Also the PEtab pysb test suite? I am worried about the failure in e.g. https://github.com/AMICI-dev/AMICI/runs/5219969341?check_suite_focus=true. If caching silently leads to wrong model equations, this is a severe issue. Unless we have a good explanation of why that happened there and know that it won't happen again, I'd keep it out of there.